### PR TITLE
ball moves, bounces, and resets

### PR DESCRIPTION
**Pull request by chrisyandev**

### CODE REVIEW

**Comment by imnorookie** [comment]
uhh shit uhh what does this do? uhhhmmm hmm.. uhh *types into keyboard* LGTM!

File: Assets/Scripts/Ball.cs [hunk 1]
```diff
@@ -4,15 +4,45 @@
 
 public class Ball : MonoBehaviour
 {
-    // Start is called before the first frame update
+    public float speed = 10;
```
Comment (le-rich) [inline Assets/Scripts/Ball.cs]: Public variables should be Capitalized, missing .<x>f at end of the variable.

Comment (ParthCv) [inline Assets/Scripts/Ball.cs]: Ayo this dude really made a public variable!!??

File: Assets/Scripts/Ball.cs [hunk 2]
```diff
@@ -4,15 +4,45 @@
...
+
+    private Rigidbody rb;
+    private Vector3 lastFrameVelocity;
+
     void Start()
     {
-        
+        rb = GetComponent<Rigidbody>();
```
Comment (le-rich) [inline Assets/Scripts/Ball.cs:15]: Getcomponents are good on awake if the component is on itself that way you dont need to rewrite it every time on Enable, ok here for velocity.

**Review by le-rich — CHANGES_REQUESTED** [review]
Naming convention, but otherwise good.

**Review by le-rich — APPROVED** [review]

**Pull request merged**